doc/source/getting_started/10min.rst

```diff
@@ -67,20 +67,20 @@ will be completed:
    @verbatim
    In [1]: df2.<TAB>  # noqa: E225, E999
    df2.A                  df2.bool
-   df2.abs                df2.boxplot
+   df2.abs()              df2.boxplot
```

Comment:
I can't check this, but are you sure we actually provide parentheses?  That seems inconsistent with the other ones.

Reply:
Oh, you are right!

sorry for this mistake.